packages/wrangler/src/deployment-bundle/node-compat.ts

```diff
@@ -124,3 +117,13 @@ export function getNodeCompatMode({
 		nodejsCompatV2,
 	};
 }
+
+/**
+ * The nodejs_compat_v2 flag currently requires an `experimental:` prefix within Wrangler,
```

Comment:
"currently" will no more be current in a few weeks/months - change the wording?

Reply:
This whole function will disappear when it is no longer current.

Reply:
> This whole function will disappear when it is no longer current.

Maybe add a `//TODO: ...` then - bonus point if there is a link to a tracking issue :)

Reply:
I even preflighted a PR!
https://github.com/cloudflare/workers-sdk/pull/6545